components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -94,7 +94,7 @@
  @warning This API will eventually be deprecated. Please use
  @c useCurrentContentSizeCategoryWhenApplied instead.
 */
-@property(nonatomic, readonly) BOOL mdc_adjustsFontForContentSizeCategory;
+@property(nonatomic, readonly) BOOL mdc_adjustsFontForContentSizeCategory __deprecated;
```

Comment:
Please use the _msg variant of the deprecated annotation and include the recommended alternative API.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/8312/commits/db318be4d8ddafcc7fbe6abaa3f14dc83eb6f7d4.